gym_solo/core/test_termination_factory.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from gym_solo.core import termination
+
+# TODO: Move this to gym_solo.testing
+class DummyTermination(termination.Termination):
+  def __init__(self, body_id: int, termination_var: bool):
+    self.body_id = body_id
+    self.termination_var = termination_var
+    self.reset_counter = 0
+    self.reset()
+    
+  def reset(self):
+    self.reset_counter += 1
+
+  def is_terminated(self) -> bool:
+    return self.termination_var
+
+class TestTerminationFactory(unittest.TestCase):
+  def test_initialization(self):
+    termination_factory = termination.TerminationFactory()
+    self.assertFalse(termination_factory._terminations)
```

Comment:
Might want to replace with `self.assertListEqual(termination_factory._terminations, [])`, as per the [documentation](https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.assertListEqual). Typically, you want to use the assertion that is the "closest" to your specific scenario.

For example, `assertFalse` will get the job, but if the assertion ever fails, the error message will be something like `Expected False but got True`. Meanwhile, if we were to use `assertListEqual`, we would get an error message like `Lists aren't equal [<TimeBasedTermation @memory_address>] != []`, which is just so much more descriptive. 

This is especially useful when we make a big refactor. It's nice when the error messages _literally_ tell you how to fix the problem! 😝

Reply:
Ahhh!! Thanks for the pro tip. I will start doing this from now on